### PR TITLE
feat: add inactive topic policies command for topic

### PR DESCRIPTION
**Pull request by nodece**
Signed-off-by: Zixuan Liu <nodeces@gmail.com>

### Changes

background from #246,  the PR implements the following commands:

- `pulsarctl topics get-inactive-topic-policies <topic> --applied <bool>`
- `pulsarctl topics remove-inactive-topic-policies <topic>`
- `pulsarctl topics set-inactive-topic-policies <topic> --enable-delete-while-inactive <bool> --max-inactive-duration <string> --delete-mode <delete_when_no_subscriptions|delete_when_subscriptions_caught_up>`

### CODE REVIEW

File: pkg/ctl/topic/get_inactive_topic.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package topic
+
+import (
+	"github.com/streamnative/pulsarctl/pkg/cmdutils"
+
```
Comment (zymap) [inline pkg/ctl/topic/get_inactive_topic.go:22]: remove the empty line?

File: pkg/ctl/topic/inactive_topic_test.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/streamnative/pulsarctl/pkg/pulsar/utils"
+	"github.com/streamnative/pulsarctl/pkg/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInactiveTopicCmd(t *testing.T) {
```
Comment (zymap) [inline pkg/ctl/topic/inactive_topic_test.go:31]: We also need to add some failure case tests.

**Review by zymap — APPROVED** [review]

File: pkg/ctl/topic/inactive_topic_test.go (hunk 2 above)
Comment (nodece) [reply pkg/ctl/topic/inactive_topic_test.go:31]: I think we just need to test the flag requires for ` set-inactive-topic-policies`.

Comment (zymap) [reply pkg/ctl/topic/inactive_topic_test.go:31]: The failure case is used to check the error response is parsed correctly at the pulsarctl

Comment (nodece) [reply pkg/ctl/topic/inactive_topic_test.go:31]: Do you want to test the response from pulsar?

**Review by zymap — APPROVED** [review]

**Pull request merged**